user_guides/pom.xml

```diff
@@ -16,84 +16,34 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>jakarta.tck</groupId>
-        <artifactId>jakarta.tck.user-guide</artifactId>
-        <version>11.0.0-SNAPSHOT</version>
-        <relativePath>parent/pom.xml</relativePath>
+        <artifactId>project</artifactId>
+        <version>10.0.0-SNAPSHOT</version>
```

Comment:
looks like we still have a lot of 10.0.0-SNAPSHOT references since we have already started adding EE 11 tests for the SPEC Ballots, we may as well shift ahead to 11.0.0-SNAPSHOT but that doesn't have to be done in this pull request.

Reply:
see #1236